pyanaconda/ui/gui/spokes/network.py

```diff
@@ -333,22 +376,16 @@ def __init__(self, builder, client, network_module, spoke=None):
         self.button_apply_hostname = self.builder.get_object("button_apply_hostname")
         self.button_apply_hostname.connect("clicked", self.on_apply_hostname)
 
+        self._wireless_dialog = SelectWirelessNetworksDialog(self.spoke.data, self.client)
```

Comment:
Why the dialogs cannot be created locally as needed as usual?

Reply:
I think they can, updated.